AlignedCollectionViewFlowLayout/Classes/AlignedCollectionViewFlowLayout.swift

```diff
@@ -84,6 +85,8 @@ public class AlignedCollectionViewFlowLayout: UICollectionViewFlowLayout {
                 return nil
             }
             return AlignmentAxis(alignment: HorizontalAlignment.right, position: collectionViewWidth - sectionInset.right)
+        case .center:
+            return AlignmentAxis(alignment: HorizontalAlignment.center, position: sectionInset.left)
```

Comment:
I get why you're defining the layout axis like this but I find it confusing for two reasons:

1. I think of an alignment axis as a vertical axis with respect to which the items are aligned.
    - For a `.left` alignment the axis is on the left.
    - For a `.right` alignment the axis is on the right.
    - Thus, for a `.center` alignment it should be in the center in order to stick to the same semantic meaning.

2. The returned alignment axis is _not_ the alignment axis that is actually used for aligning the cells. You're instantiating a new axis inside the function `horizontalCenterAlignmentAxis(for layoutAttributes: UICollectionViewLayoutAttributes)`. This is not consistent and distributes the construction of alignment axes over two different places, making the code more difficult to read.

    You should consider replacing the computed property `alignmentAxis` with a function

        func alignmentAxis(for layoutAttributes: UICollectionViewLayoutAttributes? = nil) -> AlignmentAxis<HorizontalAlignment>

    You only need to pass layout attributes for the `.center` alignment so in all other cases you just don't pass a parameter. This way, we'll still have the whole alignment axis logic in one central place.